docs/manual/manual_circuit.md

```diff
@@ -1456,11 +1456,11 @@ print(circ.is_symbolic())   # returns True when free_symbols() is non-empty
 ```
 
 ```{note}
-There are some minor drawbacks associated with symbolic compilation. When using [Euler-angle equations](https://tket.quantinuum.com/api-docs/passes.html#pytket.passes.EulerAngleReduction) or quaternions for merging adjacent rotation gates, the resulting angles are given by some lengthy trigonometric expressions which cannot be evaluated down to just a number when one of the original angles was parameterised; this can lead to unhelpfully long expressions for the angles of some gates in the compiled circuit. It is also not possible to apply the {py:class}`pytket.passes.KAKDecomposition` pass to simplify a parameterised circuit, so that pass will only apply to non-parameterised subcircuits, potentially missing some valid opportunities for optimisation.
+There are some minor drawbacks associated with symbolic compilation. When using [Euler-angle equations](inv:#*.EulerAngleReduction) or quaternions for merging adjacent rotation gates, the resulting angles are given by some lengthy trigonometric expressions which cannot be evaluated down to just a number when one of the original angles was parameterised; this can lead to unhelpfully long expressions for the angles of some gates in the compiled circuit. It is also not possible to apply the {py:class}`pytket.passes.KAKDecomposition` pass to simplify a parameterised circuit, so that pass will only apply to non-parameterised subcircuits, potentially missing some valid opportunities for optimisation.
 ```
 
 ```{seealso}
-To see how to use symbolic compilation in a variational experiment, have a look at our [VQE (UCCSD) example](https://tket.quantinuum.com/user-guide/examples/algorithms_and_protocols/ucc_vqe.html).
```

Comment:
need to fix

Reply:
done in https://github.com/CQCL/pytket-docs/pull/392/commits/875f6c79f87942040797139ad5474c48867c04f4